src/plugins/plugin.tooltip.js

```diff
@@ -16,7 +16,7 @@ const positioners = {
 	 * Average mode places the tooltip at the average position of the elements shown
 	 * @function Chart.Tooltip.positioners.average
 	 * @param items {object[]} the items being displayed in the tooltip
-	 * @returns {object} tooltip position
+	 * @returns {object} tooltip position. false if no position
```

Comment:
The return type is outdated, not sure if we want to keep these up to date now that we have types?

Reply:
No, we can probably remove the docs